clients/lightrider/lightrider-entry.js

```diff
@@ -52,19 +52,29 @@ async function runLighthouseInLR(connection, url, flags, lrOpts) {
   }
 
   try {
-    const results = await lighthouse(url, flags, config, connection);
-    if (!results) return;
+    const runnerResult = await lighthouse(url, flags, config, connection);
+    if (!runnerResult) return;
 
     if (logAssets) {
-      await assetSaver.logAssets(results.artifacts, results.lhr.audits);
+      const assetsToLog =
```

Comment:
this refactor doesn't seem to be a win. if anything, logAssets() should call its own private prepare method, but due to other comments, i dont think we'll end up needing this code anyhow.